microsoft-edge/devtools-guide-chromium/storage/indexeddb.md

```diff
@@ -28,110 +28,133 @@ To view and change [IndexedDB](https://developer.mozilla.org/docs/Web/API/Indexe
+IndexedDB keys and values aren't editable from the **Application** tool.  However, since DevTools has access to the page context, you can run JavaScript code within DevTools to edit the data stored in an IndexedDB database.
+
+#### Edit IndexedDB data by using the Console tool
+
+1. In DevTools, on the Activity Bar, select the **Console** tab.
 
-### Edit IndexedDB data with Snippets
+1. In the **Console** tool, run JavaScript code to edit the IndexedDB data. For example, to add a new value to the `keyval` object store, run the following code:
+   
+   ```javascript
+   let connection = indexedDB.open("keyval-store", 1);
 
-[Snippets](../javascript/snippets.md) are a way to store and run blocks of JavaScript code within DevTools.  When you run a Snippet, the result is logged to the **Console**.  You can use a Snippet to run JavaScript code to edit an IndexedDB database.
+   connection.onsuccess = e => {
+     const database = e.target.result;
+     const transaction = database.transaction("keyval", "readwrite");
+     const objectStore = transaction.objectStore("keyval");
+     const request = objectStore.add({foo: "bar"}, "new-key");
+     request.onsuccess = e => {
+       console.log(e.target.result);
+     }
+   }
+   ```
 
-![Using a Snippet to interact with IndexedDB](./indexeddb-images/storage-sources-snippets-indexeddb-output.png)
+#### Edit IndexedDB data by using snippets
+
+[Snippets](../javascript/snippets.md) are a way to store and run JavaScript code repeatedly, within DevTools.  If you need to edit IndexedDB data often, store it in a new snippet, and then run the snippet. To learn more, see [Run snippets of JavaScript on any webpage](../javascript/snippets.md).
+
+![Using a Snippet to interact with IndexedDB](./indexeddb-images/edit-from-snippet.png)
 
 
 <!-- ====================================================================== -->
 ## Delete IndexedDB data
 
-### Delete an IndexedDB key-value pair
+
+You can delete any of the following:
+* An IndexedDB key-value pair.
+* All key-value pairs in an object store.
+* An IndexedDB database.
+* All IndexedDB storage.
+
+These options are described below.
+
+
+#### Delete an IndexedDB key-value pair
 
 1. [View an IndexedDB object store](#view-indexeddb-data).
 
-1. Click the key-value pair that you want to delete.  DevTools highlights it to indicate that it is selected.
+1. Click the key-value pair that you want to delete.  DevTools highlights the key-value pair to indicate that it is selected:
 
-   ![Click a key-value pair in order to delete it](./indexeddb-images/storage-application-storage-indexeddb-notes_db-notes_os2.png)
+   ![A key-value pair item is selected in the object store view](./indexeddb-images/select-keyval.png)
 
-1. Press `Delete` or click **Delete Selected** (![Delete Selected](./indexeddb-images/delete-icon.png)).
+1. Press `Delete` on your keyboard or click the **Delete Selected** (![Delete Selected icon](./indexeddb-images/delete-icon.png)) button:
```

Comment:
```suggestion
1. Press the `Delete` key, or click the **Delete Selected** (![Delete Selected icon](./indexeddb-images/delete-icon.png)) button:
```
@captainbrosset - is tooltip "Delete Selected", or "Delete selected"?  Some strings in this tool have changed to sentence case.
* comma to separate the two phrases / two ways.
* I'd write just "Press Delete" - that's an alternative that's less verbose.